docs/css/style.css

```diff
@@ -0,0 +1,206 @@
+html {
+  box-sizing: border-box;
+  overflow-y: scroll;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+html, body {
+    height: 100%;
+}
+
+body {
+    font-family: "Helvetica", "Helvetica Neue", "Lucida Sans Unicode", "Arial", "sans-serif";
+    font-size: 14px;
+    color: #3a4454;
+}
+
+body, ul, h1, p {
+    margin: 0;
+    padding: 0;
+}
+
+ul {
+    list-style: none;
+}
+
+
+.logo {
+    background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBHZW5lcmF0b3I6IEFkb2JlIElsbHVzdHJhdG9yIDE5LjIuMSwgU1ZHIEV4cG9ydCBQbHVnLUluIC4gU1ZHIFZlcnNpb246IDYuMDAgQnVpbGQgMCkgIC0tPgo8c3ZnIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiIHdpZHRoPSI4MHB4IiBoZWlnaHQ9IjgwcHgiCgkgdmlld0JveD0iMCAwIDgwIDgwIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCA4MCA4MDsiIHhtbDpzcGFjZT0icHJlc2VydmUiPgo8c3R5bGUgdHlwZT0idGV4dC9jc3MiPgoJLnN0MHtkaXNwbGF5Om5vbmU7ZmlsbDojMTYxOTE5O30KCS5zdDF7ZmlsbDojNjVDMUMyO30KCS5zdDJ7ZmlsbDojRkZGRkZGO30KPC9zdHlsZT4KPGcgaWQ9IkViZW5lXzEiPgoJPHJlY3QgY2xhc3M9InN0MCIgd2lkdGg9Ijc5LjkiIGhlaWdodD0iNzkuOSIvPgoJPGcgaWQ9IkxvZ28iPgoJCTxwYXRoIGlkPSJ0cmlhbmdsZSIgY2xhc3M9InN0MSIgZD0iTTIyLjEsNjUuNWwtMTEsMTFsLTExLTExTDIyLjEsNjUuNXoiLz4KCQk8cGF0aCBpZD0iaGVyZSIgY2xhc3M9InN0MiIgZD0iTTM4LjIsNDAuNWMtMi4zLTIuNy0yLjItNC4yLTAuOC01LjZjMS43LTEuNywzLjUtMSw1LjUsMC45TDM4LjIsNDAuNXogTTYxLjgsMTAuNQoJCQljMS43LTEuNywzLjUtMSw1LjUsMC45bC00LjcsNC43QzYwLjIsMTMuNCw2MC4zLDEyLDYxLjgsMTAuNXogTTc2LDE1LjNjLTIuMSwzLjItNS45LDguNC05LjgsNC41bDkuOS05LjlDNzUuMiw5LDc0LjYsOC4yLDc0LDcuNwoJCQljLTUuMy01LjMtMTEuMy01LjQtMTYtMC43Yy0zLjEsMy4xLTQsNi43LTMuMSwxMC4xbC0zLjEtMy41QzUxLDE0LDQ3LjIsMTcuNCw1MCwyMi40bC0zLjUtMi45bC00LjcsNC43bDYuNCw2LjQKCQkJYy00LjktMy45LTEwLjItMy41LTE0LjQsMC43Yy00LjUsNC41LTQuMiwxMC0wLjgsMTQuNWwtMC43LTAuN2MtNC41LTQuNS05LjQtMi45LTExLjgtMC41Yy0xLjksMS45LTMsNC40LTIuNSw2LjJsLTkuNy05LjcKCQkJbC01LjIsNS4ybDE5LjIsMTkuMmgxMC4zbC02LjktNi45Yy0zLjYtMy43LTMuNy01LjYtMS45LTcuNGMxLjctMS43LDMuNy0wLjYsNy4yLDIuOGw2LjgsNi44bDUuMS01LjFsLTYuNS02LjUKCQkJYzQuNywzLjUsMTAuMywzLjcsMTUuMy0xLjNjMCwwLDAuMS0wLjEsMC4xLTAuMWwwLDBjMy4xLTIuOCw0LTUuNiw0LTUuNmwtMy45LTIuNmMtMi4xLDMuMi01LjgsOC40LTkuNyw0LjVsOS45LTkuOWw2LjIsNi4yCgkJCWw1LjQtNS40bC03LjctNy43Yy0zLjYtMy42LTEuNS03LDAtOC4zYzAuNywxLjUsMS43LDIuOSwzLDQuMmM1LDUsMTEuNCw2LDE3LjEsMC40YzAsMCwwLjEtMC4xLDAuMS0wLjFsMCwwYzMuMS0yLjgsNC01LjYsNC01LjYKCQkJTDc2LDE1LjN6Ii8+Cgk8L2c+CjwvZz4KPGcgaWQ9IkViZW5lXzIiPgo8L2c+Cjwvc3ZnPg==');
```

Comment:
Storing background image in-line here and loading another image from file later on line 42 does not look consistent. Why not using the same approach everywhere?